### ISSUE TITLE
Expose All the new (cached) metadata  attributes via Graph QL

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#20209]
We need to be able to gain access to the new Basic Metadata attributes via GraphQL
The attributes can be found in the Enum BasicMetadataFields
and they are 
"name","title","path","contentType","sha256","isImage","length","fileSize","modDate","width","height"
And they need to be mapped to a `GraphQLOutputType` apparently that should be done in the class `CustomFieldType`

**Comment by fabrizzio-dotCMS** [comment]
here's an example to verify:
```

FileAssetCollection(limit:20){
    fileAsset{
      modDate
      sha256
      mime
      title
      versionPath
      mime
      focalPoint
      path
      name
      versionPath
      width
      height
      size
      isImage
    }
    metaData {
        key
        value
    }    
  }
```

**Comment by alfredo-dotcms** [comment]
![image](https://user-images.githubusercontent.com/37185433/119739967-4b925e00-be40-11eb-8600-94bc941b635f.png)

**Comment by bryanboza** [comment]
Fixed, tested on release-21.06 and now we are able to access to those information from graphQL
